content/docs/iac/concepts/resources/providers.md

```diff
@@ -517,23 +542,22 @@ final var myresource = new MyResource("myResource",
 
 ## Disabling Default Providers
 
-While default providers are enabled by default, they [can be disabled](/docs/concepts/config#special-configuration-options) on a per stack basis. Disabling default
-providers is a good idea if you want to ensure that your providers must be explicitly configured and should never use the default system configuration. (The meaning of "default system configuration" depends on the provider: it may be environment variables which can differ between environments, or a configuration file in a default location, and so on.)
+While default providers are enabled by default, they [can be disabled](/docs/concepts/config#special-configuration-options) on a per stack basis. Disabling default providers is a good idea if you want to ensure that your providers must be explicitly configured and should never use the default system configuration. (The meaning of "default system configuration" depends on the provider: it may be environment variables which can differ between environments, or a configuration file in a default location, and so on.)
```

Comment:
Mention the common mistake of not passing the right resource option to set the explicit provider and not noticing the error until it's too late?